### PR TITLE
Head idempotent: Schedule errata-cache and channel-repo data just after sync repo

### PR DESCRIPTION
**Pull request by meaksh**
Currently, seems `hoag-dummy` package installation is failing for `ssh-minion` because `errata-cache` and `channel-repodata` have not been executed yet (they run automatically every minute)

This PR will explicitly schedule these tasks after syncing repository.

As a gift, this PR gets rid of `debrand_string` helper as discussed on #259

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -52,6 +52,8 @@ def debrand_string(str)
         when "Step 1: Create Kickstart Profile" then str
         when "Create Kickstart Profile" then str
         when "Test Erratum" then str
+        when "errata-cache-default" then str
```
Comment (MalloZup) [inline features/support/env.rb]: @meaksh  we want to get rid of debrand string, look at https://github.com/SUSE/spacewalk-testsuite-base/issues/259

Comment (MalloZup) [inline features/support/env.rb]: robo

Comment (meaksh) [inline features/support/env.rb]: LOL!

**Comment by meaksh** [comment]
As you said, Rubocop is happy again!

robo

**Comment by MalloZup** [comment]
once rubocop is done, let's merge 🐼

**Pull request merged**